src/libraries/Native/Unix/System.Globalization.Native/pal_locale.c

```diff
@@ -46,7 +46,7 @@ int32_t GetLocale(const UChar* localeName,
     {
         UChar c = localeName[i];
 
-        if (c > (UChar)0x7F)
+        if (c > (UChar)0x7F || c == (UChar)0x2F)
```

Comment:
Could you please use '/' instead instead of the ordinal 0z2F?

Also, could you please put a comment that '/' in some versions of ICU can cause issue it is used as resource separator? or something similar.

Reply:
Updated and re-pushed.